browser/misc_metrics/misc_android_metrics.h

```diff
@@ -15,6 +15,9 @@ class SearchEngineTracker;
 
 namespace misc_metrics {
 
+inline constexpr char kBraveCoreIsDefaultHistogramName[] =
```

Comment:
if we have no plans to use this constant externally, then perhaps we should just move it into an anonymous namespace within `misc_android_metrics.cc`